File: Assets/Tests/KotlinTokenizer/lambdas.swift


userService.updateUser(picture: picture).always { [weak self] in
    self?.hasPhoto = true
}

userService.autoLinkTenant(tenantId: tenant.id).then { [weak self] _ in
    self?.startPayment(paymentMethod, true)
}.catchError { [weak self] _ in
    let intent = self?.coordinator.autoRegisterIntent(tenant: tenant, onComplete: {
        self?.startPayment(paymentMethod, true)
    })
    self?.navigationManager.show(intent, animation: .push)
}

item.selectCallback = { option in
    presenter.selectPaymentMethod(option)
}

item.selectCallback?(option)
item.selectCallback!(option)

ints.map {
    if $0 == 0 {
        return "zero"
    }
    return "non zero"
}

File: Assets/Tests/KotlinTokenizer/lambdas.kt
userService.updateUser(picture = picture).always {  
    this.hasPhoto = true
}
userService.autoLinkTenant(tenantId = tenant.id).then { _  -> 
    this.startPayment(paymentMethod, true)
}.catchError { _  -> 
    val intent = this.coordinator.autoRegisterIntent(tenant = tenant, onComplete = { this.startPayment(paymentMethod, true) })
    this.navigationManager.show(intent, animation = .push)
}
item.selectCallback = { option  -> 
    presenter.selectPaymentMethod(option)
}
item.selectCallback?.invoke(option)
item.selectCallback!!.invoke(option)
ints.map {
    if (it == 0) {
        return@map "zero"
    }
    "non zero"
}
